frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -110,6 +130,33 @@ class CreateAccountForm extends Component {
     });
   };
 
+  redirectToLoginPage = () => {
+    // refresh the page in order to show the login form
+    window.location.reload();
+    // close dialog
+    this.setState({ showDialog: false });
+  };
+
+  handleSubmit = event => {
+    this.props
+      .registerAction({
```

Comment:
This would be a good time to ensure this redux action is tested.

Reply:
This registration API will be tested in our API tests.
See PR #299.